tests/DCPS/MultiDPTest/subscriber.cpp

```diff
@@ -128,6 +128,8 @@ void init_dcps_objects(int i)
       inst->load(ach, sect_key);
       config->instances_.push_back(inst);
       TheTransportRegistry->bind_config(config_name, participant[i]);
+
+      ACE_OS::sleep(2); // delay to ensure rtps associations complete
```

Comment:
@mitza-oci I'm still not sure why the RTPS discovery has a delay in safety profile and why the delay is 1 second. I'm curious what's the reason?

Reply:
There is no built-in delay of 1 second, however participant discovery can take some time depending on how it's configured.  Ideally these tests would have no assumptions about exactly how quickly asynchronous events like discovery take to complete.

I think last time we discussed this the plan was to make another change before merging, is that correct?

Reply:
If I heard it right, Chip is checking if that timing assumption is an issue and worth a more extensive fixing.

Other than that, the CI is having a bunch of errors, but I think that's because this branch isn't up-to-date with the master.